osu.Game.Tests/Visual/Multiplayer/TestScenePlaylistsRoomSettingsPlaylist.cs

```diff
@@ -111,18 +106,8 @@ public void TestNextItemSelectedAfterExternalDeletion()
             AddAssert("item 0 is selected", () => playlist.SelectedItem.Value == playlist.Items[0]);
         }
 
-        [Test]
-        public void TestChangeBeatmapAndRemove()
```

Comment:
Test removed because it is invalid - `PlaylistsSongSelect` no longer mutates an existing `PlaylistItem` instance and instead creates a new one every time.